frontend/src/app/modules/fields/edit/field-types/select-edit-field.component.ts

```diff
@@ -39,7 +39,7 @@ import {SelectAutocompleterRegisterService} from "app/modules/fields/edit/field-
 
 export interface ValueOption {
   name:string;
-  $href:string | null;
+  $href:string|null;
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/e1c1b00c5a6c973508c7538a865d197276aa638c/frontend/src/app/modules/fields/edit/field-types/select-edit-field.component.ts#L44"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>